Update add-private-registries.adoc

tsmithv11 opened the pull request.

tsmithv11 committed:
Update add-private-registries.adoc

anaghapamidi committed:
Tested the images in Dita and have no errors.

anaghapamidi committed:
Modified the note

anaghapamidi commented:
@tsmithv11 -I have modified the note please have a look. I have also checked the images and do not see any errors. I have restaged the content to prevent any further issues.

tsmithv11 committed:
Update add-private-registries.adoc

tsmithv11 commented on code-security/admin_guide/get-started/connect-your-repositories/add-private-registries.adoc:
```diff
@@ -6,6 +6,8 @@
 Integrating Private Registries with Prisma Cloud enables you to build an accurate dependency tree of the packages in your private registries. Using an *Artifactory* for integration ensures you have visibility into packages of your private registry and fix suggestions for vulnerabilities found in indirect packages.
 To remediate vulnerabilities found on Private registries see results and fix suggestions on *Projects*. Alternatively, you can make an informed decision by seeing the scan results on *Supply Chain*, where the graph displays the dependency tree based on the integration.
 
+NOTE: Currently, Prisma Cloud Code Security supports one Artifactory integration for every package manager and is limited to Maven, Gradle, NPM, Yarn, and Pip.
```
```suggestion
NOTE: Currently, Prisma Cloud Code Security supports one Artifactory integration, one registry per package manager and is limited to Maven, Gradle, NPM, Yarn, and Pip.
```

tsmithv11 commented:
> @tsmithv11 -I have modified the note please have a look. I have also checked the images and do not see any errors. I have restaged the content to prevent any further issues.

@anaghapamidi that edit didn't quite capture what we need to say. I added a suggestion.

Pubs-MV committed:
Update code-security/admin_guide/get-started/connect-your-repositories/add-private-registries.adoc

Co-authored-by: Taylor <28880387+tsmithv11@users.noreply.github.com>
